documentation/modules/upgrading/proc-upgrade-cluster-operator.adoc

```diff
@@ -42,14 +44,12 @@ Wait for the rolling updates to complete.
 kubectl get po my-cluster-kafka-0 -o jsonpath='{.spec.containers[0].image}'
```

Comment:
@tombentley - we do the check on the Kafka pod for the CO upgrade?

Reply:
It was something that @michalxo mentioned when reviewing the procedure. He wondered why we don't do the verification on the CO pod. 

```
kubectl get pod <cluster-operator-pod-name> -o jsonpath='{.spec.containers[0].image}'
        quay.io/strimzi/operator:latest%
```

Reply:
Still not understanding this. `get po` and `get pod` are the same thing. Or are you referring to the `quay.io/strimzi/operator:latest%` bit?

Reply:
Looking at my comment in Jira, I think the latter is what I meant to suggest for a change ->
``` wrong pod used in command?. As we upgraded strimzi CO and not kafka (that did not change)

$ kubectl get pod <cluster-operator-pod-name> -o jsonpath='{.spec.containers[0].image}'
        quay.io/strimzi/operator:latest%
```

Reply:
IMHO the upgrade of the operator is successful after all operand pods roll to the new images. So checking the image in the Kafka pods makes more sense than checking it in the Cluster Operator pod.

Reply:
Ah, finally I understand what's being asked here! (Sorry if I was being dense). Yes, checking on the operand pods makes more sense.

Reply:
@PaulRMellor If you do any edits to this, I think using the full name - `kubectl get pod` makes more sense in documentation.

Reply:
@tombentley @scholzj @michalxo  -- Sorry for the confusion and thanks for the comments. I've left the commands as is, but changed to `pod` rather than `po`